include/dxc/dxcapi.internal.h

```diff
@@ -16,9 +16,9 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 // Forward declarations.
-typedef interface ITextFont ITextFont;
-typedef interface IEnumSTATSTG IEnumSTATSTG;
-typedef interface ID3D10Blob ID3D10Blob;
+typedef struct ITextFont ITextFont;
```

Comment:
Do we still need to make this change?

Reply:
Yes, we no longer do `#define struct interface` in WinAdapter.h as it causes compilation conflicts with other usages of `interface` in other files.